tests/suit_update/tests/01-run.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import signal
+import subprocess
+import sys
+import time
+from testrunner import run
+
+UPDATE_TIMEOUT = 120
+MANIFEST_TIMEOUT = 15
+
+CURRENT_DIR = os.path.dirname(sys.argv[0])
+RIOT_DIR = os.path.abspath(os.path.join(CURRENT_DIR, '../../..'))
```

Comment:
The RIOTBASE directory can be directly retrieved using os.getenv('RIOTBASE')

Reply:
I actually don't use these variables anymore hahaha, will remove.